content/tutorials/node/simple-chat.md

```diff
@@ -1,12 +1,121 @@
 ---
-title: Build a chat App using event streams
+title: Build a Chat Application Using Fluvio Node.js Client
 toc: true
 ---
 
-TODO: Add step by step instructions on how to install and run desktop Chat App.
 
-## Section
-...
+This guide provides an example of using `@fluvio/client` to write a chat application using Node.js.
 
-### Sub-section
-...
\ No newline at end of file
+## Prerequisites
+
+Before starting on this tutorial, you'll need to have completed the following
+
+- Install [Node.js](#check-nodejs) (**v12.11.0** or above);
+- Have the Fluvio CLI installed and have access to a Fluvio cluster. See our [getting started] guide;
+- Clone the `fluvio-demo-apps-node` repository;
+
+[getting started]: /docs/getting-started
+
+### Cloning the Repository
+
+```bash
+git clone https://github.com/infinyon/fluvio-demo-apps-node.git
+```
+
+Once cloned, navigate to the `chat-app` directory by running the following command:
+
+```bash
+cd ./fluvio-demo-apps-node/chat-app
+```
+
+All commands will be run from the `chat-app` directory in this repository.
+
+## Application Overview
+
+This demo consists of a WebSocket proxy server relaying messages from a React client web application to the `@fluvio/client`, sending `user` and `chat` topic events to the Fluvio cluster.
+
+`@fluvio/client` is currently not supported in the browser, therefore the WebSocket server is required to proxy the client events. An example of this proxy service can be viewed at [`./chat-server/src/fluvio-lib/dataStream/index.ts`](https://github.com/infinyon/fluvio-demo-apps-node/blob/master/chat-app/chat-server/src/fluvio-lib/dataStreams/index.ts#L26).
+
+Read the [API docs](https://infinyon.github.io/fluvio-client-node/) for more information.
+<hr/>
```

Comment:
Please remove <hr /> as it does not display well. We can change the CSS to add more spacing across the website if that's what you intend here.

Reply:
commit b673c10 fixes.